runtime-image/jdk17-overrides.yaml

```diff
@@ -0,0 +1,55 @@
+schema_version: 1
+
+name: "jboss-eap-7-tech-preview/eap74-openjdk17-runtime-openshift-rhel8"
+description: "The JBoss EAP 7.4 OpenJDK 17 runtime image"
+from: "registry.redhat.io/ubi8/ubi:latest"
+
+labels:
+    - name: "com.redhat.component"
+      value: "jboss-eap-74-openjdk17-runtime-openshift-rhel8-container"
+
+envs:
+     - name: DISABLE_LEGACY_SECURITY
+       value: "true"
+     - name: AB_JOLOKIA_OFF
```

Comment:
It seems from discussions that we are not disabling Jolokia for this release.

Reply:
+1 updated.